src/main/java/model/Tag.java

```diff
@@ -26,6 +26,14 @@ public String getTagName() {
     public void editTagName(String newTagName) {
         this.tagName = newTagName;
     }
+    public boolean cardInTag(CardUUID cardUUID) {
```

Comment:
Function name should be a verb?

Reply:
Issues resolved in edab2bc2b42c131e9ab63358f987ffae5538ebed